src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1351,6 +1351,19 @@ public static bool IsValidExtensionMethodThisArgConversion(Conversion conversion
                 case ConversionKind.Boxing:
                 case ConversionKind.ImplicitReference:
                     return true;
+
+                case ConversionKind.ImplicitTuple:
+                case ConversionKind.ImplicitTupleLiteral:
+                    // check if all element conversions satisfy the requirement
+                    foreach (var elementConversion in conversion.UnderlyingConversions)
```

Comment:
I can never remember which ImmutableArray extensions we have and which we are still missing, but if we have it, consider `conversion.UnderlyingConversions.All(c => IsValidExtensionMethodThisArgConversion(c))`.